workloads/imagenet/imagenet_jax/workload.py

```diff
@@ -41,28 +41,6 @@ def __init__(self):
     self.dataset = 'imagenette'
```

Comment:
Thinking more about it, perhaps it makes more sense to have the Imagenette as a separate workload (instead of as a debug option in the ImageNet workload).
I am not familiar with the Imagenette one, but could it perhaps be a workload for the `development` workload class?

Reply:
I'm also not familiar with the "imagenette" dataset, [it seems to be](https://www.tensorflow.org/datasets/catalog/imagenette) an easy to classify subset of the reguar ImageNet dataset with only 10 classes. Having it as a separate `development` workload sounds reasonable to me. If we would integrate it directly in this workload, it should probably be done more cleanly than it currently is done in the Jax workload.

Reply:
Sounds good. Let's remove the imagenette dataset and keep it as an option as a `developement` workload.

Reply:
Yes, remove it from JAX workload.

You could also keep this for a separate PR. As mentioned (in some other comment) we probably need a "Cleaning up" PR anyway. This PR already includes a lot of changes to the JAX workload as is.
I will add imagenette as a possible development workload to our spreadsheet so we don't forget.